hhnk_threedi_plugin/gui/checks/sqlite_check_popup.py

```diff
@@ -116,6 +117,11 @@ def setupUi(sqlite_dialog):
     sqlite_dialog.isolated_channels_chk.setObjectName("isolated_channels_chk")
     sqlite_dialog.grid_chk = QCheckBox("Genereer grid")
     sqlite_dialog.grid_chk.setObjectName("grid_chk")
+    sqlite_dialog.cross_section_chk = QCheckBox("Profile controle")
+    sqlite_dialog.cross_section_chk.setObjectName("cross_section_chk")
+    sqlite_dialog.cross_section_intersection_chk = QCheckBox("Profiel snijpunt in vertex")
```

Comment:
Duidelijker beschrijving
Profiel snijpunt in vertex -> Cross-section op channel vertex